packages/validation/src/user-guards.ts

```diff
@@ -3,3 +3,8 @@ import { UserResponse, ExternalAuthorResponse } from '@asap-hub/model';
 export const isInternalUser = (
   author: ExternalAuthorResponse | UserResponse,
 ): author is UserResponse => (author as UserResponse).email !== undefined;
+
+export const isExternalUser = (
+  author: ExternalAuthorResponse | UserResponse,
+): author is ExternalAuthorResponse =>
+  isInternalUser(author as ExternalAuthorResponse) === false;
```

Comment:
Why do we need this? Isn't it `!isInternalUser`

Reply:
not quite, it's a typescript thing

Reply:
this validation is casting as ExternalUser not UserResponse what I need is the UserResponse + href